.github/scripts/ci.sh

```diff
@@ -5,19 +5,41 @@ set -eu -o pipefail
 
 DIR="${PWD}"
 
-curl --silent --show-error --location https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh | bash -s -- --fabric-version "$1"
+# for fabric 2.5 we don't test for BFT
+if [[ $1 == '2.5.3' && $3 == 'BFT' ]]; then
+    exit 0
+fi
 
+curl --silent --show-error --location https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh | bash -s -- --fabric-version "$1"
 cd fabric-samples/test-network
-if [[ $2 == 'create_channel' ]]; then
-    ./network.sh up
-else    
-    ./network.sh up createChannel -c mychannel
+
+echo $1
+echo $3
+
+if [[ $1 == '3.0.0-preview' && $3 == 'BFT' ]]; then 
```

Comment:
Perhaps golang.yaml could avoid invoking this script with FABRIC_VERSION=2.5.3 and CONSENSUS=BFT so this logic can be simplified to just checking for BFT?